### DISCUSSION TITLE
Can't install or update Flowise on Replit

### DISCUSSION THREAD

**Discussion by jewilhel** [FlowiseAI/Flowise#396] [General]
After the latest 1.2.13 release, I can't update or even install a fresh copy of Flowise running on Replit with Node.js

Apparently, Flowwise now requires a newer version of Node.js that Replit supports by default in its templates.
I've tried to update Node.js and then update or reinstall Flowise without success.

These are the directions I originally used to get Flowise running on Replit:

https://docs.flowiseai.com/deployment/replit

**Comment by jewilhel** [comment]
Here is the error I'm getting:

~/Flowise$ npm install -g flowise
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'flowise@1.2.13',
npm WARN EBADENGINE   required: { node: '>=18.15.0' },
npm WARN EBADENGINE   current: { node: 'v18.12.1', npm: '8.19.2' }

**Comment by jewilhel** [comment]
I've tried all sorts of ways to update Node.js to 18.15.0  in Replit before trying to install the latest version of Flowise. Nothing seems to work.... can anyone hear please help me?

**Reply by HenryHengZJ** [reply]
Hey sorry just saw this. Does it work if you install v1.2.12?

`npm install -g flowise@1.2.12`

**Reply by jewilhel** [reply]
Yes it does... I had it working with the previous version of Flowise before the Node.js requirements changed....Replit has an older template version of Node.js, and I can't find a way to update it properly.

**Reply by HenryHengZJ** [reply]
hey @jewilhel updated docs here - https://docs.flowiseai.com/deployment/replit

use `yarn add global flowise --ignore-engines` and set 3 secrets variables to bypass chromium downloads

**Reply by jewilhel** [reply]
I followed the instructions carefully but it didn't work until I realize the there was a typo in the instructions

Step 6.
"Type in npm install -g flowise into the Shell terminal window. If you are having error about incompatible node version, use the following command yarn global add react --ignore-engines"

The yarn command should be what you typed above:
yarn add global flowise --ignore-engines

not
yarn global add react --ignore-engines

**Reply by jewilhel** [reply]
Also shouldn't the first Secret have a value of true and not 1? (maybe it doesn't matter)
<img width="532" alt="Screenshot 2023-06-29 at 4 41 47 PM" src="https://github.com/FlowiseAI/Flowise/assets/8344852/6b76739c-1cac-45a3-b54f-677d85680fbe">

**Comment by jewilhel** [comment]
Thanks for your help, finally got it working.
